warehouse/models/mart/gtfs/fct_daily_scheduled_trips.sql

```diff
@@ -75,7 +75,22 @@ fct_daily_scheduled_trips AS (
         stop_times_grouped.trip_last_arrival_sec,
         stop_times_grouped.service_hours,
         stop_times_grouped.contains_warning_duplicate_primary_key AS contains_warning_duplicate_stop_times_primary_key,
-        stop_times_grouped.contains_warning_missing_foreign_key_stop_id
+        stop_times_grouped.contains_warning_missing_foreign_key_stop_id,
+
+        CASE
+            WHEN (stop_times_grouped.trip_first_departure_sec > 86400) THEN DATE_ADD(service_index.service_date, INTERVAL 1 DAY)
+            ELSE service_index.service_date
+        END AS activity_date,
+
+        CASE
+            WHEN (stop_times_grouped.trip_first_departure_sec > 86400) THEN TIME(TIMESTAMP_SECONDS(stop_times_grouped.trip_first_departure_sec - 86400))
+            ELSE TIME(TIMESTAMP_SECONDS(stop_times_grouped.trip_first_departure_sec))
```

Comment:
And I think we want this to be a datetime rather than just a `TIME`? 🤔 So maybe use `activity_date` here to construct it as well?

Reply:
As we discussed, can disregard this